README.md

```diff
@@ -45,11 +45,40 @@ includes more details on these features.
 
 ## Try Elyra
 
- You can try out some of Elyra features using the [My Binder](https://mybinder.readthedocs.io/en/latest/) service.
+#### Using Binder
+You can try out some of Elyra features using the [My Binder](https://mybinder.readthedocs.io/en/latest/) service.
 
- Click on the link below to try Elyra, on a sandbox environment, without having to install anything.
+Click on the link below to try Elyra, on a sandbox environment, without having to install anything.
 
- [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/tree/binder-demo)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/tree/binder-demo)
+
+#### Using Docker
+
+You can also try Elyra via docker.
+
+The command below starts a clean Elyra environment:
+
+```
+docker run -it -p 8888:8888 elyra/elyra:1.0.0rc3 jupyter lab --debug
+```
+
+To make your Notebook work directory available to your Docker container, 
```

Comment:
Since I'm not familiar with JupyterLab locations I thought initially this was referring to some sort of JupyterLab default directory  `${HOME}/opensource/jupyter-notebooks/`. To avoid any potential confusion I would suggest rewording the content slightly or changing the directory name to make it more clear that this can be any local directory name, such as `/path/to/local/directory`:

"To make a local directory (e.g. `${HOME}/opensource/jupyter-notebooks/`) available to your Docker container,  you can use a mount command similar to the following:"